frontend/src/routes/person/[id].svelte

```diff
@@ -64,7 +66,19 @@
                     </figure>
                     <div class="max-w-md card-body">
                         <h2 class="card-title">{person.name}</h2>
-                        <p>{person.biography}</p>
+                        {#if currentBiographyLength <= INITIAL_BIOGRAPHY_LENGTH && person.biography.length > INITIAL_BIOGRAPHY_LENGTH}
+                            <p>{person.biography.slice(0, currentBiographyLength)}...</p>
+                            <div class="flex space-x-1 justify-center p-1">
+                                <button
+                                    on:click={() => currentBiographyLength = person.biography.length}
+                                    type="button"

```

Comment:
Maybe something else than a button? A link like shown a bit higher in the issue

Reply:
See latest image, for the newest change. I will add this on movie and tv as well :D